packages/common-all/src/utils.ts

```diff
@@ -594,6 +580,26 @@ export class ConfigUtils {
       : ConfigUtils.getPreview(config).enablePrettyRefs;
   }
 
+  /**
+   * NOTE: _config currently doesn't have a `global` object. We're keeping it here
+   * to make using the API easier when we do add it
+   */
+  static getEnableChildLinks(
+    _config: IntermediateDendronConfig,
+    opts?: { note?: NoteProps }
+  ): boolean {
+    if (
+      opts &&
+      opts.note &&
+      opts.note.config &&
+      opts.note.config.global &&
+      !_.isUndefined(opts.note.config.global.enableChildLinks)
+    ) {
+      return opts.note.config.global.enableChildLinks;
+    }
+    return true;
+  }
```

Comment:
I think it's a bit confusing to introduce the global config namespace for just this key when we haven't introduced the namespace yet. 

I personally would have added them into the `publishing` and `preview` namespace separately (and explicitly) and then have the config util assume those namespaces' defaults instead.

For the users: there will be cases where they want separate control, but having a global namespace that is implied here like so would have clashing mental models and user intent.

For migration when we eventually introduce the global namespace: we want things in global to be separately available in each respective config namespaces as well. So it makes more sense to introduce them now as such rather than with an assumed global namespace. This way when the global namespace is introduced: 1. we don't have to posthumously add `enableChildLinks` to the two other namespaces (it's already there), and 2. we simply introduce the global namespace and what we have here will be naturally supported.